integration-cli/docker_cli_daemon_plugins_test.go

```diff
@@ -179,6 +179,13 @@ func (s *DockerDaemonSuite) TestDaemonShutdownWithPlugins(c *check.C) {
 	if out, ec, err := runCommandWithOutput(cmd); ec != 1 {
 		c.Fatalf("Expected exit code '1', got %d err: %v output: %s ", ec, err, out)
 	}
+
+	if err := s.d.Start("--live-restore"); err != nil {
+		c.Fatalf("could not start daemon with live restore: %v", err)
+	}
+	cmd = exec.Command("pgrep", "-f", pluginProcessName)
+	out, _, err := runCommandWithOutput(cmd)
```

Comment:
if `pgrep` exits with a non-zero exit code, then `err` will always be non-nil. The exit code comes from `err`.